code/modules/admin/verbs/adminjump.dm

```diff
@@ -16,7 +16,7 @@
 				continue
 			turfs.Add(T)
 
-	var/turf/T = pick_n_take(turfs)
+	var/turf/T = safepick(turfs)

```

Comment:
safepick isn't the same as pick_n_take, the latter remove from the list the thing it picked.